scripts/bump-version.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $SCRIPT_DIR/..
+
+OLD_VERSION="${1}"
+NEW_VERSION="${2}"
+
+echo "Current version: ${OLD_VERSION}"
+echo "Bumping version: ${NEW_VERSION}"
+
+for pkg in {dart,flutter,logging,dio}; do
+  # Bump version in pubspec.yaml
+  perl -pi -e "s/^version: .*/version: $NEW_VERSION/" $pkg/pubspec.yaml
```

Comment:
Can we do the very same, outside of the `for` loop, for the `flutter/example/pubspec.yaml` file as well?